contrib/devtools/tag-binaries.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
As said the tagging (with [<petname>-]<commit_id>) seems the important part to me, but I wouldn't oppose to archive if more people prefer it.